packages/eslint-config-next/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "eslint-config-next",
```

Comment:
Can we make sure to publish this globally to npm under `eslint-config-next` and not underneath the `@next` scope?

Reply:
Yeah will coordinate with @Timer as he has access.

Reply:
It can now be published from our CI.